SOURCES/etc/xapi.d/plugins/hyperthreading.py

```diff
@@ -12,20 +12,11 @@
 from xcpngutils import configure_logging, run_command
 
 
-# returns a JSON dict {<poolname>: {mountpoint: <mountpoint>, ...}}
-# xe host-call-plugin host-uuid=<UUID> plugin=zfs.py fn=list_zfs_pools
 def get_hyperthreading(session, args):
-    try:
-        result = run_command(['xl', 'info', 'threads_per_core'])
-        _LOGGER.info(result)
-        lines = result['stdout'].splitlines()
-
-        return json.dumps(int(lines[0]) > 1)
-    except OSError as e:
-        if e.errno == errno.ENOENT:
-            return json.dumps({})
-        else:
-            raise e
```

Comment:
Is there a reason for the `try`/`expect` to be removed?

Reply:
yes, it was a leftover copy/paste from another plugin. Here it makes no sense. If the plugin is installed and the `xl` is not present we have a big problem and we need to investigate. So I let the error through.